discounts/rust/product-discounts/fixed-amount/src/main.rs

```diff
@@ -0,0 +1,205 @@
+use serde::{Deserialize, Serialize};
+
+mod api;
+use api::*;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Configuration {
+    pub value: f64,
+}
+
+impl Configuration {
+    const DEFAULT_VALUE: f64 = 50.00;
+
+    fn from_str(str: &str) -> Self {
+        serde_json::from_str(str).expect("Unable to parse configuration value from metafield")
```

Comment:
It may be best to avoid the variable name `str` even though the compiler allows it